### PR TITLE
Update stream-label / structured-metadata assignments from Loki Operator

### PR DESCRIPTION
**Pull request by xperimental**
This updates the table with the state of the implementation from grafana/loki#14410.

### CODE REVIEW

**Comment by xperimental** [comment]
PTAL @alanconway @jcantrill @periklis

File: cluster-logging.md [hunk 1]
```diff
@@ -54,32 +54,32 @@ The "Location" column can be one of the following:
...
-| `k8s.daemonset.name` | resource | container | structured metadata | Conditionally forwarded based on creator of Pod |
-| `k8s.deployment.name` | resource | container | structured metadata | Conditionally forwarded based on creator of Pod |
-| `k8s.job.name` | resource | container | structured metadata | Conditionally forwarded based on creator of Pod |
+| `k8s.pod.uid` | resource | container | structured metadata | |
+| `k8s.cronjob.name` | resource | container | stream label | Conditionally forwarded based on creator of Pod |
+| `k8s.daemonset.name` | resource | container | stream label | Conditionally forwarded based on creator of Pod |
+| `k8s.deployment.name` | resource | container | stream label | Conditionally forwarded based on creator of Pod |
+| `k8s.job.name` | resource | container | stream label | Conditionally forwarded based on creator of Pod |
 | `k8s.replicaset.name` | resource | container | structured metadata | Conditionally forwarded based on creator of Pod |
 | `k8s.statefulset.name` | resource | container | structured metadata | Conditionally forwarded based on creator of Pod |
```
Comment (xperimental) [inline cluster-logging.md:84]: I think we forgot to make these stream labels as well? Do you agree @periklis ?

Comment (JoaoBraveCoding) [reply cluster-logging.md:84]: Seems to make sense to also make `k8s.statefulset.name` a stream label 👍

**Review by JoaoBraveCoding — APPROVED** [review]
lgtm

File: cluster-logging.md (hunk 1 above)
Comment (periklis) [reply cluster-logging.md:84]: Let's get them in then.

**Review by periklis — APPROVED** [review]

File: cluster-logging.md (hunk 1 above)
Comment (xperimental) [reply cluster-logging.md]: Done

**Review by JoaoBraveCoding — APPROVED** [review]
lgtm

**Pull request merged**